test cases/common/264 implementation/meson.build

```diff
@@ -0,0 +1,7 @@
+project('meson.implementation()',
+  default_options: {
+    'backend': 'null',
+  },
```

Comment:
Is this an optimization at all?

Reply:
The backend is intentionally simpler, doesn't write out a build.ninja, or do nearly as much processing work etc.

But I'm not actually sure the testsuite can handle that as it'll probably try to build?

Reply:
Actually, what happens here is that we unconditionally pass `--backend ninja`. :)

Reply:
I will remove this then.